test/migration/setup_runner.go

```diff
@@ -16,14 +16,14 @@ import (
 )
 
 const (
-	ProvisionedUser             = "migration-provisioned"
-	DeactivatedUser             = "migration-deactivated"
-	BannedUser                  = "migration-banned-provisioned"
-	AppStudioProvisionedUser    = "migration-appstudio-provisioned"
-	SecondMemberProvisionedUser = "migration-second-member-provisioned-user"
-
-	ProvisionedAppStudioSpace    = "migration-appstudio-provisioned-space"
-	SecondMemberProvisionedSpace = "migration-second-member-provisioned-space"
+	ProvisionedUser             = "mgr-provisioned"
+	DeactivatedUser             = "mgr-deactivated"
+	BannedUser                  = "mgr-banned"
+	AppStudioProvisionedUser    = "mgr-appstudio"
+	SecondMemberProvisionedUser = "mgr-member2-user"
+
+	ProvisionedAppStudioSpace    = "mgr-appstudio-space"
+	SecondMemberProvisionedSpace = "mgr-member2-space"
```

Comment:
mgr is usually used as an abbreviation for different words. 
how about using `mig-` prefix? And we could also shorten the other parts of the name so we can afford adding the `sp-` prefix.
```suggestion
	ProvisionedUser             = "mig-prov"
	DeactivatedUser             = "mig-deact
	BannedUser                  = "mig-banned"
	AppStudioProvisionedUser    = "mig-appst"
	SecondMemberProvisionedUser = "mig-m2-user"

	ProvisionedAppStudioSpace    = "mig-appst-space"
	SecondMemberProvisionedSpace = "mig-m2-space"
```